### ISSUE TITLE
Container with Valheim Plus disabled falsly identifies updates with new Vaheim and/or V+ and updates/restarts server every check.

### ISSUE THREAD

**Issue by RyeNCode** [lloesche/valheim-server-docker#500]
Container with Valheim Plus disabled falsly identifies updates with new Vaheim and/or V+) and updates/restarts server every check.

Log may help identify issues,
```
Sep 27 19:00:04 /supervisord: valheim-updater DEBUG - [37] - No players connected to Valheim server
Sep 27 19:00:04 /supervisord: valheim-updater INFO - Downloading/updating/validating Valheim server from Steam
Sep 27 19:00:09 /supervisord: valheim-updater Redirecting stderr to '/home/valheim/Steam/logs/stderr.txt'
Sep 27 19:00:09 /supervisord: valheim-updater [  0%] Checking for available updates...
Sep 27 19:00:09 /supervisord: valheim-updater [----] Verifying installation...
Sep 27 19:00:09 /supervisord: valheim-updater Steam Console Client (c) Valve Corporation - version 1663887295
Sep 27 19:00:09 /supervisord: valheim-updater -- type 'quit' to exit --
Sep 27 19:00:09 /supervisord: valheim-updater Loading Steam API...OK
Sep 27 19:00:09 /supervisord: valheim-updater
Sep 27 19:00:09 /supervisord: valheim-updater Connecting anonymously to Steam Public...
Sep 27 19:00:11 /supervisord: valheim-updater Message type 9806 wasn't declared, but we registered job CClientJobEnableOrDisableDownloads to handle it?
Sep 27 19:00:11 /supervisord: valheim-updater OK
Sep 27 19:00:11 /supervisord: valheim-updater Waiting for client config...OK
Sep 27 19:00:11 /supervisord: valheim-updater Waiting for user info...
Sep 27 19:00:26 /supervisord: valheim-updater OK
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x3) reconfiguring, progress: 0.00 (0 / 0)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 4.93 (62879499 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 7.50 (95779950 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 18.61 (237507047 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 25.63 (327122708 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 54.06 (690083788 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater  Update state (0x5) verifying install, progress: 82.36 (1051317612 / 1276555653)
Sep 27 19:00:26 /supervisord: valheim-updater Success! App '896660' fully installed.
Sep 27 19:00:26 /supervisord: valheim-updater *deleting   unity.lock
Sep 27 19:00:26 /supervisord: valheim-updater .d..t...... ./
Sep 27 19:00:26 /supervisord: valheim-updater INFO - Valheim Server was updated - restarting
Sep 27 19:00:26 /supervisord: valheim-updater DEBUG - [37] - ValheimPlus currently disabled but previously installed - keeping installation fresh
Sep 27 19:00:26 /supervisord: valheim-updater DEBUG - [37] - If this is unwanted remove /opt/valheim/plus
Sep 27 19:00:27 /supervisord: valheim-updater DEBUG - [2674] - Local ValheimPlus archive is identical to remote archive and was successfully installed - no update required
Sep 27 19:00:27 /supervisord: valheim-updater INFO - Valheim dedicated server or ValheimPlus mod got updated - extracting and merging installation files

```

**Comment by MotherGinger** [comment]
Experiencing this as well...

Wonder if it's related to the new crossplay updates

**Comment by themightylc** [comment]
Any obvious workaround here? Can we disable those checks?

**Comment by RyeNCode** [comment]
I've used the UPDATE_CRON (I think) env var to reduce the frequency of update checks to one day and at a known time when I know it won't be used.
That way I still get the functionality intended without the service interruption. Not great, but a mitigation of the issue for sure.
-- 
Ryan
***@***.***

**Comment by TrepnickKev** [comment]
Seems to be fixed with the latest Valheim server version 211.8.

**Comment by RyeNCode** [comment]
> Seems to be fixed with the latest Valheim server version 211.8.

Came in to say the same. Closing as
